### ISSUE TITLE
[Issue]: The playback speed can't be synchronized among group members with Syncplay

### ISSUE THREAD

**Issue by PuzzleCrab** [jellyfin/jellyfin#11302]
### Please describe your bug

I want to set the playback speed to 1.5x with Syncplay feature, but it can't work well. The playback will stuck a serveral seconds and the playback speed will be automatically pulled back to 1.0x. It cause I can't share my media with a faster playback speed. 

### Reproduction Steps

1. create a Syncplay group
2. ensure the number of members>2
3. try to set the playback speed with arbitrary user

### Jellyfin Version

10.8.13

### Environment

```markdown
- OS:Windows 10
- Linux Kernel:none
- Virtualization:
- Clients:Browser
- Browser:Firefox 91
- FFmpeg Version:
- Playback Method:Transcode
- Hardware Acceleration:none
- GPU Model:none
- Plugins:
- Reverse Proxy:
- Base URL:
- Networking:
- Storage:
```

### Jellyfin logs

```shell
None
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by jumoog** [comment]
The playback speed feature is not implemented in Syncplay, so it falls back to 1.0x.

implemented features are:

- Play
- Unpause
- Pause
- Stop
- Seek
- Shuffle
- NextItem
- PreviousItem

**Comment by pierrelasse** [comment]
*Still not implemented. Issue persists*

Related: https://features.jellyfin.org/posts/2048/syncplay-support-playback-speed-synchronization
